src/index.ts

```diff
@@ -87,13 +101,15 @@ class PluginHttp extends EventEmitter {
     this._sendIlpDestination = !!outgoing.sendIlpDestination
 
     this._incomingAuth = new Auth({
-      jwtSecret: incoming.secret,
-      staticToken: incoming.secretToken
+      jwtSecret: incoming.secret || incoming.jwtSecret,
+      staticToken: incoming.secretToken || incoming.staticToken
     })
+
+    this._useBearerToken = !outgoing.secret || !outgoing.secretToken
```

Comment:
this will send `Bearer ` if we've specified `secretToken` but not `secret`. It should only send `Bearer ` if staticToken is set.

Maybe instead you could do

```js
  staticToken: incoming.staticToken ? `Bearer ${incoming.staticToken}` : incoming.secretToken
```

and then you never have to add on the `Bearer ` at request time